Common/tests/Helpers/SimpleResultTable.cs

```diff
@@ -51,11 +51,11 @@ public Task Init(CancellationToken cancellationToken)
   public Task<IEnumerable<ResultStatusCount>> AreResultsAvailableAsync(string              sessionId,
                                                                        IEnumerable<string> keys,
                                                                        CancellationToken   cancellationToken = default)
-    => Task.FromResult<IEnumerable<ResultStatusCount>>(new ResultStatusCount[]
-                                                       {
-                                                         new(ResultStatus.Completed,
-                                                             1),
-                                                       });
+    => Task.FromResult(new List<ResultStatusCount>
```

Comment:
What is the difference in constructing the output this way?

Reply:
I changed this method so that it returns a `Ilist` then put a `IEnumerable` again and did not put back the previous implementation.
It avoid a cast from the `Task.FromResult` but `AsEnumerable` may do so too. Actually, it does not change much. I find it more readable now